orm/entities/Submission/__init__.py

```diff
@@ -15,7 +16,9 @@ class SubmissionModel(db.Model):
     submissionProofId = db.Column(db.Integer, db.ForeignKey(Proof.Model.__table__.c.proofId), nullable=False)
     latestVersionId = db.Column(db.Integer, db.ForeignKey("submissionVersion.submissionVersionId"), nullable=True)
     lockedVersionId = db.Column(db.Integer, db.ForeignKey("submissionVersion.submissionVersionId"), nullable=True)
```

Comment:
Shall we add the this to the latest version too..

Reply:
This is done.